package.json

```diff
@@ -85,19 +63,20 @@
       "logout": {
         "description": "Commands to log out of an environment."
       }
-    }
+    },
+    "flexibleTaxonomy": true
   },
   "repository": "salesforcecli/plugin-login",
   "scripts": {
     "build": "wireit",
     "clean": "sf-clean",
     "clean-all": "sf-clean all",
-    "clean:lib": "shx rm -rf lib && shx rm -rf coverage && shx rm -rf .nyc_output && shx rm -f oclif.manifest.json",
+    "clean:lib": "shx rm -rf lib && shx rm -rf coverage && shx rm -rf .nyc_output && shx rm -f oclif.manifest.json oclif.lock",
```

Comment:
nit: we could drop this "clean:lib" from our script and just use `sf-clean` which removes files from a gitignore section:

<img width="1790" alt="Screenshot 2023-11-02 at 17 34 40" src="https://github.com/salesforcecli/plugin-login/assets/6853656/69c68d1f-992a-4a40-a651-a1e58be183a5">


gitignore template: https://github.com/forcedotcom/dev-scripts/blob/cc3efa96c7b5ae1cce02ac1228187db16bfb230d/files/gitignore#L1-L24

Reply:
I logged [W-14423655](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001dmkpgYAA/view) to track this.

I'm fine either way, no need to make this change right now 👍🏼